ofl/kodemono/METADATA.pb

```diff
@@ -0,0 +1,27 @@
+name: "Kode Mono"
+designer: "Isa Ozler"
+license: "OFL"
+category: "SANS_SERIF"
+category: "MONOSPACE"
+date_added: "2023-05-04"
+fonts {
+  name: "Kode Mono"
+  style: "normal"
+  weight: 400
+  filename: "KodeMono[wght].ttf"
+  post_script_name: "KodeMono-Regular"
+  full_name: "Kode Mono Regular"
+  copyright: "Copyright 2023 The Kode Mono Project Authors (https://github.com/isaozler/kode-mono)"
+}
+subsets: "latin"
+subsets: "latin-ext"
+subsets: "menu"
+axes {
+  tag: "wght"
+  min_value: 400.0
+  max_value: 700.0
+}
+source {
+  repository_url: "https://github.com/isaozler/kode-mono"
+  commit: "15b36a40179d3678e0fe8bff19903af09c0fc0f7"
+}
```

Comment:
Add at the end (and let a line break after stroke:"SANS_SERIF")
classification: "MONOSPACE"
stroke: "SANS_SERIF"